app/sidekiq/copay_notifications/parse_new_statements_job.rb

```diff
@@ -10,9 +10,19 @@ class ParseNewStatementsJob
     JOB_INTERVAL = Settings.mcp.notifications.job_interval
     # number of jobs to perform at next interval
     BATCH_SIZE = Settings.mcp.notifications.batch_size
+    STATSD_KEY_PREFIX = 'api.copay_notifications.json_file'
+
+    sidekiq_retries_exhausted do |_msg, ex|
+      StatsD.increment("#{STATSD_KEY_PREFIX}.retries_exhausted")
+      Rails.logger.error <<~LOG
+        CopayNotifications::ParseNewStatementsJob retries exhausted:
+        Exception: #{ex.class} - #{ex.message}
+        Backtrace: #{ex.backtrace.join("\n")}
```

Comment:
Is there any PII in the backtrace?

Reply:
I believe there is no PII. Specifically backtraces only include information regarding the file name, class name and function name and look like this:
```
./app/sidekiq/copay_notifications/new_statement_notification_job.rb:34:in `block in <class:NewStatementNotificationJob>'
./spec/sidekiq/copay_notifications/new_statement_notification_job_spec.rb:132:in `block (4 levels) in <top (required)>'
/Users/.rvm/gems/ruby-3.3.3/gems/webmock-3.23.1/lib/webmock/rspec.rb:39:in `block (2 levels) in <top (required)>'
/Users/.rvm/gems/ruby-3.3.3/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:124:in `block in run'
/Users/.rvm/gems/ruby-3.3.3/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:110:in `run'
/Users/.rvm/gems/ruby-3.3.3/gems/rspec-retry-0.6.2/lib/rspec_ext/rspec_ext.rb:12:in `run_with_retry'
/Users/.rvm/gems/ruby-3.3.3/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:37:in `block (2 levels) in setup'
```
The platform also logs backtraces, for example [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/simple_forms_api/app/services/simple_forms_api/s3/submission_builder.rb#L100-L102), so I think this is safe.